tests/adapter/dbt/tests/adapter/utils/base_utils.py

```diff
@@ -15,6 +17,15 @@
 {% endtest %}
 """
 
+macros__replace_empty_sql = """
+{% macro replace_empty(expr) -%}
+case
+    when {{ expr }} = 'EMPTY' then ''
```

Comment:
Is EMPTY something `dbt` does? I have a vague recollection of that.

Reply:
The `EMPTY` here is from @ChenyuLInx's advice in https://github.com/dbt-labs/dbt-core/issues/4725#issuecomment-1047266185

Basically, dbt seed .csv files can't produce an empty string -- they can only produce `NULL`. So we do a .csv files like this instead:
```csv
input_1,input_2,output
a,b,ab
a,EMPTY,a
EMPTY,b,b
```

After creating an intermediate model that uses the `replace_empty` macro to convert `EMPTY` to empty strings, then the intermediate model will look like:

| input_1 | input_2 | output |
|---------|---------|--------|
| a       | b       | ab     |
| a       |         | a      |
|         | b       | b      |